source/agora/test/Base.d

```diff
@@ -618,6 +618,18 @@ public class TestAPIManager
         return &this.reg;
     }
 
+    /***************************************************************************
+
+        Returns:
+            A pointer to the network registry for name registries
+
+    ***************************************************************************/
+
+    public Registry!NameRegistryAPI* getNRegRegistry ()
+    {
+        return &this.nreg;
+    }
```

Comment:
Why is it a pointer ? Wouldn't `ref` be better ?

Reply:
It's a leftover anyways, will delete.